src/app/accounts/add-earn.ts

```diff
@@ -27,6 +25,8 @@ export const addEarn = async ({
   quizQuestionId: QuizQuestionId
   accountId: AccountId /* AccountId: aid validation */
 }): Promise<QuizQuestion | ApplicationError> => {
+  const rewardsConfig = getRewardsConfig()
```

Comment:
any reason to do this change?

Reply:
not sure anymore. 

slight performance optimization?

if we have the config loaded in the function, instead of been executed at startup time, it may speed up the launch of the app

I think the drawback is on memory as a new variable will be created everytime the function is been called.